recipes/cctag/all/conanfile.py

```diff
@@ -126,10 +126,11 @@ def _patch_sources(self):
         replace_in_file(self, os.path.join(self.source_folder, "src", "CMakeLists.txt"),
                               "${OpenCV_LIBS}",
                               "opencv_core opencv_videoio opencv_imgproc opencv_imgcodecs")
-        # Remove very old CUDA compute capabilities
-        replace_in_file(self, os.path.join(self.source_folder, "CMakeLists.txt"),
-                              "set(CCTAG_CUDA_CC_LIST_INIT0 3.5 3.7 5.0 5.2)",
-                              "set(CCTAG_CUDA_CC_LIST_INIT0 5.0 5.2)")
+        if Version(self.version) < "1.0.4":
+            # Remove very old CUDA compute capabilities
+            replace_in_file(self, os.path.join(self.source_folder, "CMakeLists.txt"),
```

Comment:
Hello @EstebanDugueperoux2, the PR title describes a new `cctag` version but here you are also altering the recipe.
Could you shed some light on why it is no longer necessary to remove these files?
It is very recommended in this cases to add a comment describing why this changes are not necessary anymore.

Reply:
Hello @perseoGI I have added a comment about https://github.com/alicevision/CCTag/pull/210/files
Tell me if sufficient.